shiny/app.py

```diff
@@ -0,0 +1,121 @@
+from shiny import App, Inputs, Outputs, Session, reactive, render, ui
+from shiny.types import FileInfo
+import numpy as np, pandas as pd
+from pyreadr import read_r
+from scipy.io import loadmat
+from BootstrapReport import ObjectOfInterest
+
+app_ui = ui.page_fixed(
+    ui.div(style = "height:30px"),
+    ui.panel_title("BootstrapReport"),
+
+    ui.layout_sidebar(
+
+      ui.panel_sidebar(
+        ui.input_numeric("est", "Point estimate", 0),
+        ui.input_numeric("se", "Standard error", 1),
+        ui.input_file("repcsv", "Choose data file (CSV, MAT, DTA, RDS, RDATA, XLS, XLSX)", accept = [".csv", ".mat", ".dta", ".rds", ".rdata", ".xls", ".xlsx"], multiple = False),
```

Comment:
Hit an error with XLSX below.

@MosesStewart have you tested with this file format?

![image](https://github.com/JMSLab/BootstrapReport/assets/7840353/0bf926b1-df73-48a6-aa60-f2f396b93f70)